lua/clipboard-image/health.lua

```diff
@@ -50,9 +49,9 @@ M.check = function()
 
   health.report_start "Checking dependencies"
```

Comment:
I think maybe this line should also change to
```lua
vim.health.report_start "Checking dependencies"
```

Reply:
Definitely should be. I've got that already in my own fork, I must have missed it when editing this pull request, thank you.